src/router/console/assets.js

```diff
@@ -283,7 +283,7 @@ export default [
       {
         path: ':id/update',
         name: 'GatewayUpdate',
-        component: () => import('@/views/assets/Domain/DomainDetail/GatewayCreateUpdate.vue'),
+        component: () => import('@/views/assets/Zone/ZoneDetail/GatewayCreateUpdate.vue'),
         meta: {
           title: i18n.t('GatewayUpdate'),
           permissions: ['assets.change_gateway']
```

Comment:
I'm sorry, but I can't see any specific code changes you're referring to. Could you please clarify what exactly is causing the issue?